Resources/test-parameters.json

```diff
@@ -39,7 +39,10 @@
             "value": "not-a-real-key"
         },
         "templateBaseUrl": {
-            "value": "https://raw.githubusercontent.com/SkillsFundingAgency/dss-employmentprogression/master-v3/Resources/"
+            "value": "https://raw.githubusercontent.com/SkillsFundingAgency/dss-employmentprogression/SDD-127-AddAzureMapsAppSettings-v3/Resources/"
```

Comment:
I'll need to set this back to "master" after this is merged,  unfortunately.. it seems that updating the local nested template in step with this doesn't have a great story.